### DISCUSSION TITLE
Failed to connect NEAR wallet (Fireblocks)

### DISCUSSION THREAD

**Discussion by miaojiang** [WalletConnect/walletconnect-monorepo#4179] [Developer Support]
Hi, I am trying to connect a NEAR wallet to a Dapp. 

The Dapp uses Near's Wallet Selector, which uses Wallet Connect v2. The Dapp was able to display the QR code, but when the user scans the QR code in Fireblocks, the Dapp shows the below error.

![telegram-cloud-photo-size-1-5015017489888816014-y](https://github.com/WalletConnect/walletconnect-monorepo/assets/8129919/7ed562cc-3487-4ac1-b9af-35e9a5318b41)

What is the best way to debug this issue? Fireblocks isn't providing any useful error messages.

**Comment by glitch-txs** [comment]
please contact the Dapp as we don't have control on how they are implementing our SDKs 🙏
